apps/emqx_authz/src/emqx_authz_mnesia.erl

```diff
@@ -45,6 +50,12 @@ mnesia(boot) ->
 description() ->
     "AuthZ with Mnesia".
 
+init(Source) -> Source.
+
+destroy(_Source) -> ok.
+
+dry_run(_Source) -> ok.
```

Comment:
Question: Nothing is created here because authz_mnesia is always present?

Reply:
Yes, there is no "connection" to mnesia that we create or destroy.